src/main/java/Barry.java

```diff
@@ -0,0 +1,322 @@
+        private final String eventStartDateTime;
+        private final String eventEndDateTime;
+
+        // Constructor for Events, initialised with the name of the task, as well as the start and end of the event
+        public Event(String name, String start, String end) {
+            super(name);
+            this.eventStartDateTime = start;
+            this.eventEndDateTime = end;
+        }
+
+        // Method which returns details of the object as a String
+        @Override
+        public final String detailsAsString() {
+            String basicInfo = "[E][" + (this.getStatus() ? "X" : " ") + "] " + this.getName();
+            return basicInfo + " (from: " + eventStartDateTime + " to: " + eventEndDateTime + ")";
+        }
+    }
+
+    // ArrayList for storing user-added tasks.
+    private static final ArrayList<Task> taskList = new ArrayList<>();
+
+    // Greets the user
+    private static void greet() {
+        System.out.println("\t____________________________________________________________");
+        System.out.println("\t Greetings! I'm Barry.\n" + "\t How could I assist you?");
+        System.out.println("\t [Type help for a list of available commands]");
+        System.out.println("\t____________________________________________________________\n");
+    }
+
+    // Exits the program with a message
+    private static void exit() {
+        System.out.println("\t____________________________________________________________");
+        System.out.println("\t Bye, see you next time! :)");
+        System.out.println("\t____________________________________________________________\n");
+    }
+
+    // List all available commands
+    private static void help() {
+        System.out.println("\t____________________________________________________________");
+        System.out.println("\t Here's a list of available commands:");
+        System.out.println("\t   help                                           List all available commands");
+        System.out.println("\t   bye                                            Quit");
+        System.out.println("\t   list                                           List all tasks in your list");
+        System.out.println("\t   mark [x]                                       Mark task number x in your list");
+        System.out.println("\t   unmark [x]                                     Unmark task number x in your list");
+        System.out.println("\t   delete [x]                                     Delete task number x in your list");
+        System.out.println("\t   todo [task name]                               Add a ToDo to your list");
+        System.out.println("\t   deadline [task name] /by [deadline]            Add a Deadline to your list");
+        System.out.println("\t   event [task name] /from [start] /to [end]      Add an Event to your list");
+        System.out.println("\t____________________________________________________________\n");
+    }
```

Comment:
I like that you put the UIs into different methods but perhaps I think you could create a Ui class which encapsulates all this methods together. Also I like that you have a help menu it will really help the user out!